### ISSUE TITLE
[Bug] Build failures on Debian/kfreebsd

### ISSUE THREAD

**Issue by cbiedl** [appneta/tcpreplay#706]
See the full build log at <https://buildd.debian.org/status/fetch.php?pkg=tcpreplay&arch=kfreebsd-amd64&ver=4.4.0-1&stamp=1644098467&raw=0>:

```fuzzing.c: In function ‘fuzzing’:
fuzzing.c:314:45: error: ‘ETHERTYPE_IPV6’ undeclared (first use in this function); did you mean ‘ETHERTYPE_IP6’?
  314 |                                 (l2proto == ETHERTYPE_IPV6) ? l4data : NULL);
      |                                             ^~~~~~~~~~~~~~
      |                                             ETHERTYPE_IP6
```

To be honest, I haven't checked yet the differences in that architecture - possibly indeed just using `ETHERTYPE_IP6` was enough to fix this. From browsing the sources it seems this should be done anyway already for consistency reasons.

**Comment by fklassen** [comment]
I'll investigate Monday. If I find something I should have time In Tuesday to work on it.

**Comment by fklassen** [comment]
Fixed in PR #707
